packages/php-wasm/node/src/test/php-request-handler.spec.ts

```diff
@@ -178,6 +178,42 @@ describe.each(SupportedPHPVersions)(
 			});
 			expect(response.json).toEqual({ foo: 'bar' });
 		});
+
+		it('Can accept a request body with a size of 1MB without crashing', async () => {
+			php.writeFile('/index.php', `<?php echo 'Hello World';`);
+			const response = await handler.request({
+				url: '/',
+				body: '#'.repeat(1024 * 1024),
+			});
+			expect(response).toEqual({
+				httpStatusCode: 200,
+				headers: {
+					'content-type': ['text/html; charset=UTF-8'],
+					'x-powered-by': [expect.any(String)],
+				},
+				bytes: new TextEncoder().encode('Hello World'),
+				errors: '',
+				exitCode: 1, // @TODO This should be 0
+			});
+		});
+
+		it('Can accept a request body with a size of ~512MB without crashing', async () => {
+			php.writeFile('/index.php', `<?php echo 'Hello World';`);
+			const response = await handler.request({
+				url: '/',
+				body: '#'.repeat(1024 * 1024 * 512 + -32),
+			});
+			expect(response).toEqual({
+				httpStatusCode: 200,
+				headers: {
+					'content-type': ['text/html; charset=UTF-8'],
+					'x-powered-by': [expect.any(String)],
+				},
+				bytes: new TextEncoder().encode('Hello World'),
+				errors: '',
+				exitCode: 1, // @TODO This should be 0
```

Comment:
what's this TODO about? should we leave this as a failing test for now and make it `0` so we don't forget?

Reply:
To be perfectly honest, I'm not entirely sure. I suspect its a separate, non-blocking bug, so I've been following the existing pattern found here:

https://github.com/WordPress/wordpress-playground/blob/44c2028b06b7c3e1dbb9928eda78095e1e969cba/packages/php-wasm/node/src/test/php-request-handler.spec.ts#L35

https://github.com/WordPress/wordpress-playground/blob/44c2028b06b7c3e1dbb9928eda78095e1e969cba/packages/php-wasm/node/src/test/php-request-handler.spec.ts#L123

https://github.com/WordPress/wordpress-playground/blob/44c2028b06b7c3e1dbb9928eda78095e1e969cba/packages/php-wasm/node/src/test/php-request-handler.spec.ts#L133

Reply:
fair enough. we can resolve this in another PR some time. I didn't realize we were just copying it along.